apps/contracts/contracts/Constants.sol

```diff
@@ -9,10 +9,26 @@ contract Constants {
         RewardsDistribution
     }
 
-    uint256 public constant GOVERNANCE_CYCLE = 0;
+    // Each state by days
+    // [42, 15, 2, 1]
+    uint8 [] STATE_LENGTHS = [42, 57, 59, 60];
+
+    enum VerificationState {
+        ProposalRegistered,
+        MunicipalVerification,
+        StateVerification,
+        FederalVerification,
+        SolvedNoFundingRequired,
+        RejectProposal,
+        ApproveForFunding
+    }
+
+    uint256 public GOVERNANCE_CYCLE = 0;
 
     uint256 public constant THRESHOLD_VOTE_LIMIT = 51;
 
-    uint256 public constant GOVERNANCE_CYCLE_LENGTH = 2595000;
+    // Length of a full governance cycle in seconds
+    // 60 days = 60 * 24 * 60 * 60
+    uint256 public constant GOVERNANCE_CYCLE_LENGTH = 5184000;
```

Comment:
@shyam-patel-kira this means the full cycle is of 60 days?

Should not be the sum of the states days duration? stored in `STATE_LENGHTS`

Reply:
Yes it is

Reply:
State length array is representative of the respective state length by days

Reply:
so if u observe carefully the respective stateLengths are [42, 15, 2, 1] and the array represents the day it will start so [42, 57, 59, 60]

Reply:
This doesn't answer the question... 

> State length array is representative of the respective state length by days

So there are 4 states.
Each state has it's own days duration

So the Total Duration is = `42+57+59+60` days??

But the governance cycle length is not `42+57+59+60` days. That is what I'm asking

Reply:
I see I get the variable name might be misleading...the stateLengths are [42, 15, 2, 1] but elements in this array denotes Nth day. N being the day when the state is completed. So governance cycle length is 45+15+2+1 = 60 days but the 42 being the day when verification state is completed and 57 being the day Voting stage is concluded.

Let me know if you have any suggestions for the variable name or perhaps should I just add description for the array in comments?

Reply:
> I see I get the variable name might be misleading...the stateLengths are [42, 15, 2, 1] but elements in this array denotes Nth day. N being the day when the state is completed. So governance cycle length is 45+15+2+1 = 60 days but the 42 being the day when verification state is completed and 57 being the day Voting stage is concluded.
> 
> Let me know if you have any suggestions for the variable name or perhaps should I just add description for the array in comments?

Yeah It was making no sense. Thanks! 
As answer, do both. 

- If you quicly find a better descriptive name, change it. Optional
- If you can document the variables or wathever is needed in comments, add it please. A must

best regards !

Reply:
Done